src/components/ProductDescription/ProductDescription.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./ProductDescription.scss";
+
+const ProductDescription = () => {
+  return (
+    <div className="myContainer">
```

Comment:
Pelo o que entendi, a @brubriele havia comentado de padronizarmos os nomes das classes como `my-container`, por exemplo, ao invés de `myContainer`. Eu já fiz as alterações nos meus arquivos, mas como vamos realmente padronizar isso? Eu acho que `myContainer` fica melhor. O que vocês acham? @brubriele @LiaLobo @vanessa-ariel @anacoxta

Reply:
Eu acho que com camel case fica melhor, pq como já estamos usando o BEM, a nomeenclatura das classes fica muito extensa, e evita confundir o nome da classe quando tiver mais de uma palavra ex: my-container com uma classe modificadora, por exemplo.

Reply:
Muito boa a troca do nome do componente! Eu prefiro em camel case também, tenho mais facilidade para entender. Mas nesse caso não poderia ser somente container? E deixar padronizados os containers assim? 
Uma mudança que faria no estilo é de deixar o media querie no final arquivo, abaixo de todas as outras classes ao invés de deixar ele aninhado. 
E só uma dúvida. As classes productDescription__pricing-group e productDescription__discount-percent também não tinham que ser camel case? (se decidirmos por ficar o camel case mesmo)

Reply:
Verdade hahahaha bem observado @LiaLobo. Vou trocar pra camel case e mudar a posição do media query. Esse myContainer eu estava utilizando para testar, na verdade ele não deveria ser comitado, pq esse componente ocupa a largura inteira do componente mae, então não precisa desse container.

Reply:
Por mim tranquilo mudar o padrão camelCase, meninas! Fica de fato mais fácil de não confundir com os modificadores @vanessa-ariel ! Sobre as medias queries acho que faz mais sentido setar dentro das classes que eles pertencem/estão modificando e não ao final, mais por questão de organização e pra não ter de repetir a mesma classe ao final só para setar a mq.. Deixei marcado no review

Reply:
Beleza, @brubriele. A questão dos media queries eu to mais acostumada a usar dentro das classes que pertencem também. Eu adicionei esse tópico lá na issue de padronização pra todas decidirem amanhã, aí padronizamos em todos componentes (;